ArkKit/Ark.swift

```diff
@@ -35,6 +40,11 @@ class Ark {
         // TODO: initialize animation system
         let animationSystem = ArkAnimationSystem()
         ecsManager.addSystem(animationSystem)
+        
+        
+        // Initialize game with specified physics engine that conforms to `ark-physics-facade`
```

Comment:
Same as above, we'll probably move this elsewhere into the `blueprint` to store as a callback, then we'll run the callback within `start()` and input the callback params `ecsContext, eventContext` here